src/sentry/web/api.py

```diff
@@ -175,6 +175,7 @@ def _dispatch(self, request, helper, project_id=None, origin=None, *args, **kwar
             if not project:
                 raise APIError('Client must be upgraded for CORS support')
             if not is_valid_origin(origin, project):
+                tsdb.incr(tsdb.models.project_total_received_cors, project.id or None)
```

Comment:
If `None` actually works as a key here (and I don't think it will, but I didn't check), it's totally unintentional. I'd wrap the `incr` in the conditional instead.